src/flag_gems/utils/random_utils.py

```diff
@@ -17,6 +17,18 @@ def philox_cuda_seed_offset(increment, device=None):
     gen.set_state(state_copy)
     return seed, offset
 
+def philox_mlu_seed_offset(increment, device=None):
+    device = device or torch.mlu.current_device()
+    gen = torch.mlu.default_generators[device]
+    state_copy = gen.get_state()
+    c0, c1 = state_copy.view(torch.int64)[-2:]
```

Comment:
how many bits is state_copy? is it a list longer than 2 after viewed as torch.int64?